app/Providers/ModelServiceProvider.php

```diff
@@ -30,13 +34,23 @@ public function boot()
             }
         });
 
+         // When Posts are created reject test events.
+         Post::created(function ($post) {
+            // Check if caption matches 'Test runscope upload' and 'caption_ghost_test'                
```

Comment:
This comment feels unnecessary since it's so similar to the actual line of code it's describing. Generally, I think comments are best used to describe the "why" for a line (so, here, a good comment might describe why why we're waiting 15 minutes after the post is submitted to delete it, or why we're rejecting test posts in the first place).

Reply:
ok, gotcha!